doc/source/rllib/getting-started.rst

```diff
@@ -0,0 +1,449 @@
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. _rllib-getting-started:
+
+Getting Started
+===============
+
+.. include:: /_includes/rllib/new_api_stack.rst
+
+.. _rllib-in-60min:
+
+RLlib in 60 minutes
+-------------------
+
+.. figure:: images/rllib-index-header.svg
+
+In this tutorial, you learn how to design, customize, and run an end-to-end RLlib learning experiment
+from scratch. This includes picking and configuring an :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`,
+running a couple of training iterations, saving the state of your
+:py:class:`~ray.rllib.algorithms.algorithm.Algorithm` from time to time, running a separate
+evaluation loop, and finally utilizing one of the checkpoints to deploy your trained model
+to an environment outside of RLlib and compute actions.
+
+You also learn how to customize your :ref:`RL environment <rllib-key-concepts-environments>`
+and your :ref:`neural network model <rllib-key-concepts-rl-modules>`.
+
+Installation
+~~~~~~~~~~~~
+
+First, install RLlib, `PyTorch <https://pytorch.org>`__, and `Farama Gymnasium <https://gymnasium.farama.org>`__ as shown below:
+
+.. code-block:: bash
+
+    pip install "ray[rllib]" torch "gymnasium[atari,accept-rom-license,mujoco]"
+
+
+.. _rllib-python-api:
+
+Python API
+~~~~~~~~~~
+
+RLlib's Python API provides all the flexibility required for applying the library to any
```

Comment:
Do we have any other API than the Python one?

Reply:
Nope :D We got rid of the CLI, b/c of the maintenance burden, its stark limitations, and it being more or less a duplicate of a subset of what the python API could do.

Reply:
Well, we are working on the external access protocol for clients to connect to and communicate with RLlib, but that's heavily wip.